src/NServiceBus.NHibernate.AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -10,7 +10,7 @@ public partial class TestSuiteConstraints
         public bool SupportsDelayedDelivery => true;
         public bool SupportsOutbox => true;
         public bool SupportsPurgeOnStartup => true;
-        public IConfigureEndpointTestExecution CreateTransportConfiguration() => new ConfigureEndpointAcceptanceTestingTransport(SupportsNativePubSub, SupportsDelayedDelivery);
+        public IConfigureEndpointTestExecution CreateTransportConfiguration() => new ConfigureEndpointAcceptanceTestingTransport(SupportsNativePubSub, SupportsDelayedDelivery, TransportTransactionMode.ReceiveOnly);
```

Comment:
How about we apply the same as I did in the other persistences by letting the tests run under the default and switch the mode where it is necessary?

Reply:
👍